compiler/rustc_target/src/spec/targets/aarch64_apple_visionos.rs

```diff
@@ -0,0 +1,27 @@
+use crate::spec::base::apple::{opts, visionos_llvm_target, Arch};
+use crate::spec::{FramePointer, SanitizerSet, Target, TargetOptions};
+
+pub fn target() -> Target {
+    let arch = Arch::Arm64;
+    let mut base = opts("visionos", arch);
+    base.supported_sanitizers = SanitizerSet::ADDRESS | SanitizerSet::THREAD;
+
+    Target {
+        llvm_target: visionos_llvm_target(arch).into(),
+        metadata: crate::spec::TargetMetadata {
+            description: None,
+            tier: None,
```

Comment:
Can you set these values in both targets?

Reply:
@davidtwco What is supposed to go here? I went through many of the existing targets and did not find a single one with a defined `TargetMetadata`.

Reply:
I think the intention is to make a machine-readable form of some of the stuff in `src/doc/rustc/src/platform-support/apple-visionos.md`.

So I think this'd be something like:
```rust
TargetMetadata {
    description: Some("Apple visionOS 1.0+ on arm64".into()),
    // description: Some("Apple visionOS 1.0+ Simulator on arm64".into()),
    tier: Some(3),
    host_tools: Some(false),
    std: Some(true),
}
```

Reply:
@madsmtm Can you point me to an example of that? I didn't find anything about it, and while I also inferred that's what it was for, I don't want to randomly take a guess.

Reply:
I _was_ randomly guessing, I think we'd have to ask @Nilstrieb to get a definitive answer?

Reply:
The description is the short text you added to the summary table in the docs: "ARM64 Apple visionOS". Basically what madsmtm said.